flask/app/schemas.py

```diff
@@ -36,30 +39,37 @@ class TissueSampleSchema(SQLAlchemyAutoSchema):
     class Meta:
         model = TissueSample
         include_fk = True
+        include_relationships = True
         load_instance = False
         exclude = ("created_by_id", "updated_by_id")
 
 
-class DatasetSchema(SQLAlchemyAutoSchema):
+class RNASeqDatasetSchema(SQLAlchemyAutoSchema):
     """
     POST /api/datasets
-    - a valid tissue_sample_id is required for the dataset to be linked
-    - linked_files must be a valid list
     - created_by_id and updated_by_id is not given as part of the request body and should therefore be excluded from validation
+    - discriminator field is populated automatically when a new dataset is inserted into the database.
```

Comment:
I think it would be helpful to clarify why `RNASeqDataset` (both model and name of the class) is used here instead of `Dataset` for transparency